src/drt/src/TritonRoute.i

```diff
@@ -96,6 +96,30 @@ void detailed_route_cmd(const char* guideFile,
   router->main();
 }
 
+void pin_access_cmd(const char* dbProcessNode,
+                    const char* bottomRoutingLayer,
+                    const char* topRoutingLayer,
+                    int verbose)
+{
+  auto* router = ord::OpenRoad::openRoad()->getTritonRoute();
+  router->setParams({"",
+                    "",
+                    "",
+                    "",
+                    "",
+                    dbProcessNode,
+                    false,
+                    -1,
+                    0,
+                    0,
+                    0,
+                    0,
+                    bottomRoutingLayer,
+                    topRoutingLayer,
+                    verbose});
```

Comment:
@maliberty It's not simple because ParamStruct has const string references in it, So it's either I change them to strings in ParamStruct or leave it as it is.

Reply:
Or I can have custom constructor with only the required values in pin_access_cmd

Reply:
The `const String&` was put there initially to save memory overhead. I don't think it's significant, so do whatever's convenient.

Reply:
I would drop the reference